packages/minimongo/minimongo.js

```diff
@@ -221,14 +221,26 @@ LocalCollection.Cursor.prototype.fetch = function () {
  * @locus Anywhere
  * @returns {Number}
  */
-LocalCollection.Cursor.prototype.count = function () {
+LocalCollection.Cursor.prototype.count = function (applySkipLimit) {
   var self = this;
+  // Have to respect `applySkipLimit`, so we remove limit & skip if `applySkipLimit`
```

Comment:
If I am reading [the source correctly](https://github.com/meteor/meteor/blob/devel/packages/minimongo/minimongo.js#L440-L511), `count()` uses the results gained from `_getRawObjects`, which acts directly on `_docs`, which is an IdMap; none of these things get me anywhere near anything waiting for a response from Mongo.